src/core_pdf/impl/engine/spec/s_08_graphics/device_profiles.py

```
@@ -0,0 +1,117 @@
+) -> tuple[int, int, int]:
+    """Convert one DeviceCMYK colour given as four floats in [0, 1] to sRGB."""
+    return cmyk_to_srgb(
+        internal_component_byte(cyan),
+        internal_component_byte(magenta),
+        internal_component_byte(yellow),
+        internal_component_byte(black),
+    )
+
+
+def internal_component_byte(value: float) -> int:
+    return max(0, min(255, int(round(value * 255.0))))
```

Ruff (0.16.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)